src/components/Download/DownloadOptionsForm/AggregateOptions.js

```diff
@@ -38,7 +39,10 @@ export const AggregateOptions = ({
           value={value}
           valueKey={{ key: 'value', reduce: true }}
           margin={{ horizontal: 'xxsmall' }}
-          onChange={(e) => handleChange(name, e.value)}
+          onChange={(e) => {
+            handleUpdateDownloadOptions(name, e.value)
+            return handleChange({ target: { name, value: e.value } })
+          }}
```

Comment:
What do you think about this?

```suggestion
          onChange={({ value }) => {
            handleUpdateDownloadOptions(name, value)
            return handleChange({ target: { name, value } })
          }}
```

Reply:
Oh, that's a good point! But the reason why the event object `e` is used is because the name `value` has been already declared  (at line 8) as a prop name for this component.